scripts/data_importer_service/data_import_service.py

```diff
@@ -55,6 +56,8 @@ def process_data_file(self, path, file_extension):
         for assessment in participant_assessments:
 
             for variable in assessment:
+                assessment_variables.append(variable)
+
                 isUnsupportedValue = (
                     assessment[variable] == math.inf
                     or assessment[variable] == -math.inf
```

Comment:
Didn't realize that `math.inf` works for infinity.  Much nicer than `float("infinity")` I've been using all these years.

Reply:
yes and i like it because we are also dealing with infinity as a data type which json doesn't like